extension/background.js

```diff
@@ -509,9 +509,9 @@ function recordEvent(element, decision) {
   responses.push(new SurveySubmission.Response(
       'Response', decision['name']));
   responses.push(new SurveySubmission.Response(
-      'Details', decision['details']));
+      'Details', decision['details'].toString()));
```

Comment:
Does the SurveySubmission.Response expect the second param to be a string? Maybe you could add a check in the constructor (there and wherever applicable).